pkg/minikube/kubeconfig/settings.go

```diff
@@ -83,6 +85,14 @@ func PopulateFromSettings(cfg *Settings, apiCfg *api.Config) error {
 	} else {
 		cluster.CertificateAuthority = cfg.CertificateAuthority
 	}
+
+	lastUpdate := time.Now().String()
```

Comment:
Can this use `.Format(time.RFC3339)`?

Reply:
also it doesn't seem like assigning a variable here serves any real purpose. i don't think lastUpdate gets reused anywhere, right?

Reply:
It may help humans who edit their `~/.kube/config` to determine how fresh this section might be?

Reply:
good catch the var was initially used two places got lost in refactor.

Reply:
good call on formatting.